client/components/App.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { CSSTransitionGroup } from 'react-transition-group';
+import Card from './Card';
+import LeftArrowMain from './LeftArrowMain';
+import RightArrowMain from './RightArrowMain';
+
+// style import
+import '../style/index.scss';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeCardIndex: 1,
+      activeCard: this.props.sampleData[0],
+      cardArray: this.props.sampleData,
+    }
+  }
+
+  componentDidMount() {
+    // fetch request to populate data
```

Comment:
I don't quite understand what is happening inside componentDidMount

Reply:
I am going to complete the component with mock/sample data first. And then I will complete the lifecycle method with proper fetch methods in there.